### PR TITLE
Book of Lost Spells: More reference links

### PR DESCRIPTION
**Pull request by BeAuMaN**
Hello.

Since the last update, I figured I'd go through and track through all the conjuration and necromancy spells that referred to something else, and add those links. For Conjuration, Summon Undead and Vermin Swarm work like Conjure Beasts, so I formatted them as such with a list of links to filters with the appropriate creatures in each case. I also set it up to include Adventure NPCs and Reprinted just like the Conjure Beasts spell, but I'm not sure why those are included.

Other Conjuration spells included references to creatures or items (in the case of items, I only linked to the item if the full stats weren't given; unlike PHB, FGG has spells that summon full items (i.e. +1 Scimitar) as part of the spell): Spirit Cartographer

- Blade of Light
- Tenacious Blade
- Devouring Darkness
- Elemental Scimitar
- Hound of Hell
- Call Divine Warrior
- Crag Warrior

I also did the same for Necromancy spells:

- Corpse Armor (I changed the text from half plate +1 to +1 half plate armor since that was the link. I'm not sure what argument allows me to link to an item with custom text instead of the standard text.)
- Umbral Touch
- Transform Zombie
- Umbral Weapon
- Crew with the Dead
- Create Crypt Thing
- Sanguine Creatures

Sanguine Creatures was given a filter similar to Conjure Beasts. Create Crypt Thing creates a creature that's in FGG's Fifth Edition Foes. After loading it I noticed the reference was broken by the inclusion of the term 3pp, so I edited that. This is the only instance of a creature from Fifth Edition Foes being created by a spell; I wonder if it would be more useful to copy the creature from Fifth Edition Foes to Book of Lost Spells, so that people have no need to load Fifth Edition Foes simply for that one creature.

Hope this helps!

### CODE REVIEW

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]
Looks like you've broken the JSON somewhere--run it through https://www.jsonlint.com/

File: spell/Frog God Games; Book of Lost Spells.json [hunk 1]
```diff
@@ -3870,7 +3870,7 @@
 				]
 			},
 			"entries": [
-				"You conjure a blade of light in your hand. Treat this blade as a longsword +1, +2 vs. evil creatures. The blade emits bright light in a 20-foot radius."
+				"You conjure a blade of light in your hand. Treat this blade as a {@item longsword +1|+1 longsword|dmg}, +2 vs. evil creatures. The blade emits bright light in a 20-foot radius."
```
Comment (TheGiddyLimit) [inline spell/Frog God Games; Book of Lost Spells.json:3873]: it's `{@<tag> <5etools name>|<source>|<display name>}`, so this is broken. Should be `{@item +1 longsword}` if you want it to show up as `+1 longsword`

File: spell/Frog God Games; Book of Lost Spells.json [hunk 2]
```diff
@@ -6113,7 +6113,7 @@
 			},
 			"entries": [
 				"You summon the spirit of a powerful, once-living champion of your deity and give it physical form. The spirit champion arrives within range, looking as it did in the prime of its life. The spell must be cast at a site hallowed to the caster's deity.",
-				"The spirit champion possesses all abilities and items that it possessed during life. In addition, the spirit champion is able to cast command or suggestion as an action on any worshipper of the caster's deity. Its exact statistics are determined by the GM; we suggest using the planetar's stat block as a starting point, but the specifics are entirely up to the GM. It won't give or loan items to other characters.",
+				"The spirit champion possesses all abilities and items that it possessed during life. In addition, the spirit champion is able to cast command or suggestion as an action on any worshipper of the caster's deity. Its exact statistics are determined by the GM; we suggest using the {@creature planetar's|planetar} stat block as a starting point, but the specifics are entirely up to the GM. It won't give or loan items to other characters.",
```
Comment (TheGiddyLimit) [inline spell/Frog God Games; Book of Lost Spells.json:6116]: `{@creature planetar's|planetar}` -> `{@creature planetar||planetar's}`
note that the `|<source>|` bit can just be blank if it's a MM creature

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]
`@tags` need scrubbing up

File: spell/Frog God Games; Book of Lost Spells.json (hunk 1 above)
Comment (BeAuMaN) [reply spell/Frog God Games; Book of Lost Spells.json:3873]: Thanks for that, I'll re-do all these with the correct format!

**Review by TheGiddyLimit — APPROVED** [review]

**Pull request merged**